### ISSUE TITLE
The accuracy of TSM finetuned in UCF101

### ISSUE THREAD

**Issue by chenyihang1993** [mit-han-lab/temporal-shift-module#9]
I got how to finetune TSM in UCF101, but there is not the accuracy posted. Could you tell me the accuracy? THX.

**Comment by tonylins** [comment]
Hi, we provided the accuracy of TSM fine-tuned on UCF101 in the Table 1 of our paper. The top-1 accuracy is 95.9% using 8-frame input and 2-crop test.
